src/main/java/org/jsweet/gradle/JSweetTranspileTask.java

```diff
@@ -98,11 +98,11 @@ protected void transpile() {
             if (configuration.getFactoryClassName() != null) {
                 try {
                     factory = (JSweetFactory) Thread.currentThread().getContextClassLoader()
-                            .loadClass(configuration.getFactoryClassName()).newInstance();
+                            .loadClass(configuration.getFactoryClassName()).getDeclaredConstructor().newInstance();
```

Comment:
what does this modification and the following will change please?

Reply:
It's been deprecated since java 9 so I corrected it.
Do you prefer a dedicated PR?
https://docs.oracle.com/javase/9/docs/api/java/lang/Class.html#newInstance--